test/crds/crd_validation_test.go

```diff
@@ -118,6 +183,14 @@ func getSchema(t *testing.T, crd string) validation.Schema {
 	return schema
 }
 
+func getSchemaVersioned(t *testing.T, crd string, version string) validation.Schema {
```

Comment:
Shouldn't `func getSchema(t *testing.T, crd string) validation.Schema` method be deleted?

Reply:
There are both variants

Reply:
but it is being used?

Reply:
It's not currently being used. We can decide to keep it available for use or deleting it. I decided the former but I'm fine with both

Reply:
private method not being used => delete.

we always have git history to restore it.

Reply:
Deleted